### PR TITLE
SPRING: Update projects data

### PR DESCRIPTION
**Pull request by cgrigis**
For #136

### CODE REVIEW

**Review by ineiti — APPROVED** [review]
LGTM - I'll let you decide if you want to wait some more days before merging, or merge now and perhaps update later.

**Comment by cgrigis** [comment]
No answer yet, so I am merging now and we'll update later.

**Pull request merged**
